plugins/modules/packaging/os/pkgng.py

```diff
@@ -111,6 +118,11 @@
   pkgng:
     name: baz
     state: latest
+
+- name: Upgrade all installed packages (see warning for I(name) first!)
```

Comment:
That formatting doesn't work in `EXAMPLES`.
```suggestion
- name: Upgrade all installed packages (see warning for name first!)
```
(Maybe it's better to write `see warning for the name option first!` to make clear that the `name` option is meant.)